apps/skilavottord/web/screens/RecyclingCompanies/components/RecyclingCompanyForm/RecyclingCompanyForm.tsx

```diff
@@ -124,6 +124,15 @@ const RecyclingCompanyForm: FC<
                 }}
                 error={errors?.companyName?.message}
                 backgroundColor="blue"
+                onChange={(event) => {
+                  // User the the municipality should not be able to create his own companyId
```

Comment:
Update comment? Does this text make sense?
Need to remove duplicate "the"?
```suggestion
                  // User with the municipality role should not be able to create his own companyId
```

Reply:
Fixed